test/Helper/HeadLinkTest.php

```diff
@@ -319,18 +321,24 @@ public function testConditionalStylesheetCreationNoIEWidthSpaces()
         $this->assertStringContainsString('<!--<![endif]-->', $string);
     }
 
-    public function testSettingAlternateWithTooFewArgsRaisesException()
+    public function argumentCountProvider() : iterable
```

Comment:
Please run the coding standard check. There should be no space for the colon.

Reply:
CS checks pass - you're right of course, but for some reason it's not picked up. Coding standard 1.x is in use - perhaps another pull to update it to ~2 at some point?

Reply:
The new continuous integration workflow must be added before. A new pull request needed like this one: https://github.com/laminas/laminas-paginator/pull/18